frontend/lib/store_widgets/accepting_store_summary.dart

```diff
@@ -164,7 +164,7 @@ class StoreTextOverview extends StatelessWidget {
                 overflow: TextOverflow.ellipsis,
                 style: Theme.of(context).textTheme.bodyText1),
             const SizedBox(height: 4),
-            Text(store.description ?? "",
+            Text(store.description ?? "Keine Beschreibung verfügbar",
```

Comment:
Maybe just "Keine Beschreibung" to keep it even shorter?

Reply:
Imo `Keine Beschreibung verfügbar` is better, but on smaller devices it is cut anyway with `...`. I don't really care though, not sure who has expertise here.

Reply:
I'm fine with both so lets leave it :)